packages/extract-text2/index.js

```diff
@@ -31,8 +31,8 @@ function extractText (outputFilePattern, fileType) {
             test: context.fileType(fileType),
             exclude: loaderConfig.exclude,
             loader: plugin.extract({
-              fallbackLoader: 'style-loader',
-              loader: nonStyleLoaders
+              fallback: 'style-loader',
+              use: nonStyleLoaders
             }),
```

Comment:
OK. Feel free to reject then as this is a nit really
Where does 'process.noDeprecations = true' go?

@jvanbruegge - you seem to be everywhere :)